docs/vnext/openapi.yaml

```diff
@@ -466,20 +384,22 @@ components:
       required:
         - referenceId
       properties:
-        referenceId:
-          $ref: '#/components/schemas/ResourceReferenceId'
+        identifiers:

```

Comment:
I'm thinking that the identifiers object should be a subset of the `createOrUpdate` payload. This way converting a regular resource to an `existing` resource will only require removing non-identifier properties, but nothing will have to be moved around or renamed in Bicep. We have the same experience in Bicep for ARM resources.

Reply:
Yeah, this aligns with my understanding as well. I've included a detailed description to clearly outline this.